yscope-compose/demo-assets/clp-config.yml

```
@@ -0,0 +1,39 @@
+  host: "${REPLACE_IP}"
+  port: 6005
+  db_name: "clp-query-results"
+  stream_collection_name: "stream-files"
+webui:
+  host: "localhost"
+  port: 6000
+  logging_level: "INFO"
+log_viewer_webui:
+  host: "localhost"
+  port: 6006
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)